docs/modules/ROOT/pages/sharing_peripherals.adoc

```diff
@@ -8,6 +8,7 @@ The following examples shows different ways to use the on-board LED on a Raspber
 
 Using mutual exclusion is the simplest way to share a peripheral.
 
+TIP: The depenencies you need to run this example link:/book/dev/basic_application.html#_the_cargo_toml[can be found here].
```

Comment:
Repeated typo: "depenencies". 

Also, I would rephrase the sentence to use more generic wording:
"Dependencies needed to run this example ...

Reply:
Thanks! Fixed!